### PR TITLE
New Featured Artist: False Noise

### PR DESCRIPTION
**Pull request by Ephemeralis**
🎵

### CODE REVIEW

File: news/2019-10-23-new-featured-artist-false-noise.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <source src="https://assets.ppy.sh/artists/52/previews/1527.mp3" type="audio/mpeg">
+</audio>
+
+-------
+
+As always, the above tracks and all the others [**False Noise's** Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/52) are immediately available for modding and mapping, no fuss, and 100% free.
+
+This now leaves us with this week's newest dilemma: the ominous *hint* of our next release to come. Toy's got the reins on this one, so I'll leave it to him:
+
+*You wake up in a dark room. The pungent stench of blight and mildew fills your lungs as you fiddle around for a light switch.*
```
Comment (cl8n) [inline news/2019-10-23-new-featured-artist-false-noise.md]: if the italics is just for the quote you should use actual quote syntax instead

File: news/2019-10-23-new-featured-artist-false-noise.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+*Distant figures celebrate, chugging alcohol and blaring music.*
+
+*"What are those things? Trolls?" you mutter,  mind racing trying to recall anything about their arrival.*
+
+*You catch a green speck dart behind you. "We're better."*
+
+But what does this all mean? Spooky! Find out next week!
+
+—Ephemeral, Toy
```
Comment (cl8n) [inline news/2019-10-23-new-featured-artist-false-noise.md]:
```suggestion
—Ephemeral and Toy
```
reads a little better when "by" is put in front of it, I think

File: news/2019-10-23-new-featured-artist-false-noise.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+### Kek
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/52/previews/1527.mp3" type="audio/mpeg">
+</audio>
+
+-------
+
+As always, the above tracks and all the others are immediately available from [**False Noise's** Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/52) for modding and mapping, no fuss, and 100% free.
```
Comment (cl8n) [inline news/2019-10-23-new-featured-artist-false-noise.md]:
```suggestion
As always, the above tracks and all the others are immediately available from [**False Noise**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/52) for modding and mapping, no fuss, and 100% free.
```

File: news/2019-10-23-new-featured-artist-false-noise.md (hunk 1 above)
Comment (Ephemeralis) [reply news/2019-10-23-new-featured-artist-false-noise.md]: i'm not sure how our news parser handles blockquotes to be honest

Comment (cl8n) [reply news/2019-10-23-new-featured-artist-false-noise.md]: welp I'm not either so nvm

**Pull request merged**